<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FastForwardConfigWidget</class>
 <widget class="QWidget" name="FastForwardConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>271</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="generalSettings" native="true">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="targetNameLabel">
        <property name="text">
         <string>Target name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KComboBox" name="connectionType">
        <item>
         <property name="text">
          <string notr="true">SQLite</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">MySQL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">PostgreSQL</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="connectionTypeLabel">
        <property name="text">
         <string>Connection type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="targetName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="embeddedSettings" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="databaseLocationLabel">
        <property name="text">
         <string>Database location</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KLineEdit" name="databaseLocation"/>
      </item>
     </layout>
     <zorder>generalSettings</zorder>
     <zorder>databaseLocation</zorder>
     <zorder>databaseLocationLabel</zorder>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="externalSettings" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="3" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KLineEdit" name="username"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="password"/>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="databaseName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="databaseNameLabel">
        <property name="text">
         <string>Database name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="hostnameLabel">
        <property name="text">
         <string>Hostname</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KLineEdit" name="hostname"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="KIntSpinBox" name="port"/>
      </item>
     </layout>
     <zorder>username</zorder>
     <zorder>usernameLabel</zorder>
     <zorder>usernameLabel</zorder>
     <zorder>passwordLabel</zorder>
     <zorder>password</zorder>
     <zorder>databaseName</zorder>
     <zorder>databaseNameLabel</zorder>
     <zorder>hostnameLabel</zorder>
     <zorder>hostname</zorder>
     <zorder>portLabel</zorder>
     <zorder>port</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
